### PR TITLE
Initial implementation of wireless STA and remote mode configuration

### PR DESCRIPTION
**Pull request by afaikafk**
Saving of settings now performs a device reboot, please comment whether that's acceptable or not. The reason for going with it is because there is no backup of some of the config files modified at runtime by remotesetup, like dnsmasq config, thus a reboot restores it's original state, e.g. when going back from STA to AP mode.

### CODE REVIEW

**Comment by foxbunny** [comment]
Rebooting is fine as long as user knows that that's what's going to happen.

File: librarian_netinterfaces/config.ini [hunk 1]
```diff
@@ -19,7 +19,15 @@ templates = views
 config = /etc/hostapd.conf
 
 # Hostapd service management commands
-restart_command = service hostapd restart
+restart_command = reboot
```
Comment (foxbunny) [inline librarian_netinterfaces/config.ini:22]: Should probably only reboot if it really needs it (e.g., hostapd was not active when the settings were applied). I think the `ap` command on rxOS supports asking for status, but I could be wrong.

**Comment by afaikafk** [comment]
Ok, button has this label now: "Save and Reboot" and the success message also says that "...the device is going to reboot now."

File: librarian_netinterfaces/consts.py [hunk 2]
```diff
@@ -293,3 +293,10 @@
     (STANDARD, _('Standard (most devices)')),
     (ALTERNATIVE, _('Alternative (Realtek-based devices)'))
 )
+
+AP_MODE = 'ap'
+STA_MODE = 'sta'
+MODES = (
```
Comment (foxbunny) [inline librarian_netinterfaces/consts.py:299]: These should be 'Create a hotspot' and 'Connect to a wireless network'

**Pull request merged**